game.js

```diff
@@ -0,0 +1,112 @@
+function player(name, arr) {
+  const chances = [
+    [0, 4, 8],
+    [1, 4, 7],
+    [2, 5, 8],
+    [0, 3, 6],
+    [2, 4, 6],
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+  ];
```

Comment:
That is not something related to only one player. 
All the player will use it.

You should move it out from there. 
Define a method like `checkWinning` and declare it into as constant in order for it to be used to determine the game status.